plugins/uefi-db/fu-uefi-db-device.c

```diff
@@ -86,6 +86,49 @@ fu_uefi_db_device_write_firmware(FuDevice *device,
 	return TRUE;
 }
 
+static void
+fu_uefi_db_device_add_security_attrs(FuDevice *device, FuSecurityAttrs *attrs)
+{
+	GPtrArray *children = fu_device_get_children(device);
+	gboolean seen_old = FALSE;
+	gboolean seen_new = FALSE;
+	g_autoptr(FwupdSecurityAttr) attr = NULL;
+
+	/* look for both versions of the Microsoft UEFI CA */
+	for (guint i = 0; i < children->len; i++) {
+		FuDevice *child = g_ptr_array_index(children, i);
+		if (fu_device_has_instance_id(child,
+					      "UEFI\\CRT_A5B7C551CEDC06B94D0C5B920F473E03C2F142F2",
+					      FU_DEVICE_INSTANCE_FLAG_VISIBLE)) {
+			seen_new = TRUE;
+			break;
+		}
+		if (fu_device_has_instance_id(child,
+					      "UEFI\\CRT_03DE12BE14CA397DF20CEE646C7D9B727FCCE5F8",
+					      FU_DEVICE_INSTANCE_FLAG_VISIBLE)) {
+			seen_old = TRUE;
+			break;
+		}
+	}
+
+	/* user is using a custom UEFI db, just ignore this HSI attribute */
+	if (!seen_new && !seen_old)
+		return;
+
+	/* create attr */
+	attr = fu_device_security_attr_new(device, FWUPD_SECURITY_ATTR_ID_UEFI_DB);
+	fwupd_security_attr_set_result_success(attr, FWUPD_SECURITY_ATTR_RESULT_VALID);
+	fwupd_security_attr_add_flag(attr, FWUPD_SECURITY_ATTR_FLAG_CAN_FIX);
```

Comment:
is this really can fix?  Not all the OEMs are going to roll out PK and KEK updates are they?

Reply:
According to an inside source, the OEM reception to this has been lukewarm to say the least.

Reply:
Can we leave out `CAN_FIX` and later control that based upon OEMs actually _releasing_ updates?

Reply:
Done.